### ISSUE TITLE
make a set of tune() function

### ISSUE THREAD

**Issue by roblanf** [roblanf/netcaster#15]
For each layer type, we can tune it as follows:

1. Make and evaluate all networks that are 1-step mutations from that layer
2. Keep the best one. 

We can do the same for hyperparameters like learning rate and # epochs, and batch size of final epoch. 

This potentially provides a more effective (or at least, more thorough) method of hill climbing.
